recipes/sentencepiece/meta.yaml

```diff
@@ -0,0 +1,40 @@
+{% set name = "sentencepiece" %}
+{% set version = "0.1.85" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: https://github.com/google/{{ name }}/archive/v{{ version }}.tar.gz
+  sha256: dd4956287a1b6af3cbdbbd499b7227a859a4e3f41c9882de5e6bdd929e219ae6
+
+build:
+  number: 0
+
+requirements:
+  build:
+    - cmake
+    - {{ compiler('cxx') }}
+    - gcc
```

Comment:
The current CI error is because of `gcc`. Please remove `gcc` - to use GCC compiler, please add `{{ compiler('c') }}` instead, https://conda-forge.readthedocs.io/en/latest/meta.html#compilers.

Reply:
Fixed it now.